src/documentation/user_guides/basic.malloynb

```diff
@@ -128,7 +128,7 @@ run: airports -> {
 Sources that are defined in one file can be imported into another using `import "path/to/some/file.malloy"`. For example, if the `airports` source above were defined in a file called `flights.malloy`, you could create a new file that imports it and immediately start using the `airports` source:
 >>>markdown
 ```malloy
-import "airports.malloy" 
+import "flights.malloy" 
```

Comment:
This code actually gets run as part of the documentation build, so it's probably better to fix the text than change the code.